compiler/src/iree/compiler/Codegen/Common/test/math_transform_devicelib.mlir

```diff
@@ -0,0 +1,64 @@
+// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-codegen-math-transform))' --split-input-file %s | FileCheck %s
```

Comment:
name tests consistently with the passes (`math_transform.mlir` as the pass is called `iree-codegen-math-transform`)